opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -641,13 +629,17 @@ const buildCompleteUser = async (context, client) => {
   const allowed_organizations = await batchOrganizations(context, SYSTEM_USER, client.id, batchOpts);
   const userOrganizations = allowed_organizations.map((m) => m.internal_id);
   const isUserPlatform = settings.platform_organization ? userOrganizations.includes(settings.platform_organization) : true;
-  const capabilities = await getCapabilities(context, client.id, isUserPlatform);
   const [individuals, organizations, groups] = await Promise.all([individualsPromise, organizationsPromise, userGroupsPromise]);
+  const roles = await getRoles(context, groups);
+  const capabilities = await getCapabilities(context, client.id, roles, isUserPlatform);
+  const default_hidden_types = await getDefaultHiddenTypes(context, client.id, roles);
```

Comment:
default hidden types and capabilities could be fetched at the same time

Reply:
fixed

Reply:
they can't be resolved at the same time because capabilities is either a Promise either a ValueOfUnion... and default_hidden_types is a Promise

Reply:
You can get a promise for both and do a Promise.all